docs/source/tutorial/02-add-the-graphql-schema.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "2. Add the GraphQL schema"
+---
+
+This tutorial uses a modified version of the GraphQL server you build as part of [the Apollo full-stack tutorial](https://www.apollographql.com/docs/tutorial/introduction/). You can visit [`https://apollo-fullstack-tutorial.herokuapp.com/`](https://apollo-fullstack-tutorial.herokuapp.com/) to start it up and open the GraphiQL tool to explore its schema.
+
+<img src="images/graphiql.png" alt="The GraphiQL query explorer" class="screenshot"/>
+
+The [schema](https://graphql.org/learn/schema/) defines which GraphQL operations your server can execute. Click **Docs** on the right-hand side to view a list of types you can query (and the types of fields on those types) along with any possible mutations or subscriptions.
+
+<img src="images/graphiql_show_schema.png" alt="GraphiQL showing the schema" class="screenshot">
+</img>
+
+## Download your server's schema
+
+Apollo Android requires a schema to generate typesafe models and code from your queries. There are multiple ways to get a schema. For an exemple, you can download a json schema directly from GraphiQL by clicking **Schema > Download** in the right pane.
+
+In this tutorial, we will use the `apolloDownloadSchema` task that is created by the plugin automatically. Since GraphQL supports [introspection](https://graphql.org/learn/introspection/), this will work with any GraphQL endpoint.
+
+From the root of the project, type:
+
+```shell:title=(shell)
+mkdir -p app/src/main/graphql/com/example/rocketreserver/
+./gradlew :app:downloadApolloSchema -Pcom.apollographql.apollo.endpoint='https://apollo-fullstack-tutorial.herokuapp.com/' -Pcom.apollographql.apollo.schema='src/main/graphql/com/example/rocketreserver/schema.json'
```

Comment:
with your recent PR I guess this should be changed?

Reply:
Yes. I'm still struggling to keep the doc in sync with the releases though so that most likely won't happen until `2.1.1`

Reply:
Out of scope: Btw, would it be possible take the schema.json location from the configuration if `schemaFile` property is set?